tests/sample/reflect_packet.c

```diff
@@ -100,6 +86,166 @@ reflect_packet(xdp_md_t* ctx)
         }
     }
 
+Done:
+    return rc;
+}
+
+int
+encap_ipv4_reflect_packet(xdp_md_t* ctx)
```

Comment:
```suggestion
encapsulate_ipv4_reflect_packet(xdp_md_t* ctx)
```
avoid abbreviations per style guide